Implements Amphibian Crate

ophaq opened the pull request:
## About The Pull Request
- This adds a new crate to cargo, the "Amphibian Friends Crate!" Gives you one frog and one axolotl. 
- Edits "code\modules\cargo\packs\livestock.dm"
## Why It's Good For The Game
Great for those who want a frog or axolotl and couldn't find them in maintenance or really want an amphibian friend but someone else took it. Also helps those who want to do amphibian related cytology. Players seem to love frogs and axolotls in the game, so this also adds another method of getting them via cargo.
## Changelog
:cl:
add: Adds a new crate to cargo, the "Amphibian Friends Crate!" Gives you one frog and one axolotl.
/:cl:

ophaq committed:
Update livestock.dm

ophaq committed:
Update livestock.dm

ophaq committed:
Update livestock.dm

ophaq committed:
Update livestock.dm

ophaq committed:
Implements Amphibian Crate

Adds a new crate to cargo which gives you one frog and one axolotl.

ophaq commented:
Side note: If you see anything wrong please let me know what/how I can make it better cause this is my first PR. I triple checked everything so hopefully things go smoothly?

san7890 commented on code/modules/cargo/packs/livestock.dm:
```diff
@@ -190,6 +190,16 @@
+	desc = "Two disgustingly cute slimey friends. Cytologists love them! \
+		Contains one frog and one axolotl. Warning: Frog may have hallucinogenic properties."
+	cost = CARGO_CRATE_VALUE * 4
+	contains = list(/mob/living/basic/axolotl,
+					/mob/living/basic/frog,
+					)
```
```suggestion
	contains = list(
	/mob/living/basic/axolotl,
	/mob/living/basic/frog,
	)
```

your head was in the right place but your indentation was quite odd

ophaq replied:
No errors, nothing in my git. Ran on test server and worked fine. Absolutely no clue otherwise. I copy pasted from another crate that had exact same indentation.

san7890 replied:
i got smartphowned by github webeditor

ophaq committed:
Update code/modules/cargo/packs/livestock.dm

Melbert knows better than I do. Additional indentations commited unless told otherwise.

Co-authored-by: MrMelbert <51863163+MrMelbert@users.noreply.github.com>

DemoCrack commented:
frog
